src/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -152,8 +152,8 @@ public void SendFile_Synchronous(IPAddress listenAt, bool sendPreAndPostBuffers,
             using (client)
             {
                 client.SendFile(filename, preBuffer, postBuffer, TransmitFileOptions.UseDefaultWorkerThread);
-
                 client.LingerState = new LingerOption(true, LingerTime);
+                client.Shutdown(SocketShutdown.Send);
```

Comment:
If we Shutdown() the socket before dispose (which we should, to increase the test reliability), then there is no need to set the LingerOption